CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 ## Unreleased
 
 * Update to LUX 308 ([PR #3394](https://github.com/alphagov/govuk_publishing_components/pull/3394))
+* World Locations paths are now generated based on the base path if it is provided. If there is no base path provided, it will fall to previous behaviour of generating the slug based on page title. The special cases list has been removed.
```

Comment:
I think that this should be reframed from the perspective of the component - related navigation - both here and an in the PR

Reply:
Updated ! let me know what you think